src/elements/Progressbar/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames/bind';
+import style from './style.scss';
+import numberInRange from '../../utils/numberInRange';
+
+const classNames = cx.bind(style);
+const colorMap = {
+  success: 'color-success',
+  warning: 'color-warning',
+  error: 'color-error',
+};
```

Comment:
we don't need this map variable.

Reply:
why don't we have primary color as default, and have success, warning, and error?